src/scrape_up/github/issue.py

```diff
@@ -67,4 +68,34 @@ def title(self):
         except:
             message = "No title found"
             return message
+        
+    def opened_at(self,time_zone='UTC'):
+        """
+        Returns a string containing the time range when the issue was opened in local time
+        """
+        data = self.__scrape_page()
+        opened_time = data.find('relative-time')['datetime']
+        opened_time_utc = datetime.fromisoformat(opened_time[:-1]).replace(tzinfo=pytz.utc)
+        opened_time_local = opened_time_utc.astimezone(pytz.timezone(time_zone))
+        return self.__time_range(opened_time_local,time_zone)
+
+    def __time_range(self, date_time,time_zone):
+        """
+        Returns a string containing the time range between the given date/time and now
+        """
+        now = datetime.now(pytz.timezone(time_zone))
+        delta = now - date_time
+
+        days = delta.days
+        seconds = delta.seconds
 
+        if days > 7:
+            return f"{days // 7} weeks ago"
+        elif days > 0:
+            return f"{days} days ago"
+        elif seconds >= 3600:
+            return f"{seconds // 3600} hours ago"
+        elif seconds >= 60:
+            return f"{seconds // 60} minutes ago"
+        else:
+            return "just now"
```

Comment:
You don't have to do that much. Just scrape the text written here in an issue.
The text after, opened this issue


![image](https://user-images.githubusercontent.com/93156825/219968240-7730f2c6-c425-4b5c-83a2-f1b54ca17d53.png)

Reply:
updated

Reply:
updated